go/server/handlers.go

```diff
@@ -51,15 +51,15 @@ func getPlannerVersion(c *gin.Context) macrobench.PlannerVersion {
 	planner := macrobench.V3Planner
 	plannerStr, err := c.Cookie("vtgatePlanner")
 	if err != nil {
-		slog.Warn(err.Error())
+		// cookie is not set, then use the default
+		return planner
 	}
```

Comment:
The whole statement can be removed

Reply:
Yeah I know, but I thought making it explicit will be more readable